packages/components/src/typings.ts

```diff
@@ -1,13 +1,8 @@
+/// <reference lib="dom" />
```

Comment:
This is the magic that tells Typescript to include the dom library if it ever needs to interpret this file.